styles.js

```diff
@@ -71,6 +71,11 @@ CKEDITOR.stylesSet.add( 'default', [
 		element: 'ol',
 		attributes: { 'class': 'list-decimal' }
 	},
+	{
+		name: 'List: 1. 1. 1.',
+		element: 'ol',
+		attributes: { 'class': 'list-decimal-point' }
+	},
```

Comment:
Overlooked this file in the previous PR https://github.com/PolicyStat/ckeditor-dev/pull/31